setup.py

```diff
@@ -30,7 +30,7 @@
 
 setup(
     name='sqlalchemy_hawq',
-    version='1.0.0',
+    version='1.0.1',
```

Comment:
should the minor version number be bumped instead now that CI is setup which is a new feature?

Reply:
I went with patch since it is a feature that doesn't affect the package code at all